### ISSUE TITLE
Why is "version" a field in the submission?

### ISSUE THREAD

**Issue by arokem** [openjournals/joss#442]
Version information often changes during the review (e.g., when proposed changes are integrated into the software, and a new version is made as part of the review process). Besides, the version information doesn't make it to the final paper. Why are we even asking for it? 

See also: https://github.com/maxrousseau/pfla/issues/2

**Comment by labarba** [comment]
I agree this is awkward! It's even a required field when submitting; at least make it optional.

**Comment by jedbrown** [comment]
I think the version is counterproductive at submission time (allowing a
non-default branch would be nice), but that the post-review version to
be archived should probably be (preferably signed) tag.  Could we teach
whedon to perform consistency checks/automatically identify the version
from the DOI?

**Comment by danielskatz** [comment]
I'm going to close this, in favor of https://github.com/openjournals/joss/issues/857 where a related discussion happened.
